tutorial_list.tsv

```diff
@@ -0,0 +1,3 @@
+Tutorial	Description
```

Comment:
This file shouldn't go in the base project directory. Probably the best place for it is inst.

Reply:
moved to `inst/`